lingua_franca/res/text/en-us/normalize.json

```diff
@@ -99,7 +99,78 @@
     "you'd've": "you would have",
     "you'll": "you will",
     "you're": "you are",
-    "you've": "you have"
+    "you've": "you have",
+    "I'm'a": "I am about to",
+    "I'm'o": "I am going to",
+    "I'll've": "I will have",
+    "I'd've": "I would have",
+    "Whatcha": "What are you",
+    "amn't": "am not",
+    "'cause": "because",
+    "can't've": "cannot have",
+    "couldn't've": "could not have",
+    "daren't": "dare not",
+    "daresn't": "dare not",
+    "dasn't": "dare not",
+    "everyone's": "everyone is",
+    "gimme": "give me",
+    "gon't": "go not",
+    "hadn't've": "had not have",
+    "he've": "he have",
+    "he'll've": "he will have",
+    "he'd've": "he would have",
+    "here's": "here is",
+    "how're": "how are",
+    "how'd'y": "how do you",
+    "'tis": "it is",
+    "'twas": "it was",
+    "it'll've": "it will have",
+    "it'd've": "it would have",
+    "kinda": "kind of",
+    "let's": "let us",
+    "ma'am": "madam",
+    "may've": "may have",
+    "mayn't": "may not",
```

Comment:
This is an uncommon word. Is "mightn't" in there? That one's more common, particularly across the pond, though still not really common